hw6/docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: "2.1"
+services:
+  mariadb:
+    image: lscr.io/linuxserver/mariadb:latest
+    container_name: mariadb
+    environment:
+      - PUID=1000
+      - PGID=1000
+      - MYSQL_ROOT_PASSWORD=believe
+      - TZ=Europe/London
+      - MYSQL_USER=root #optional
+      - MYSQL_PASSWORD=believe #optional
```

Comment:
Not sure why you are putting a root and user password? Maybe might be for security reasons?

Reply:
I was unable to connect without id and password..I have updated it without exposing password in the script